backend/routes/door.go

```diff
@@ -9,6 +9,7 @@ func DoorsRoute() chi.Router {
 	r := chi.NewRouter()
 	r.Get("/", controller.GetAllDoors)
 	r.Post("/", controller.CreateDoor)
+	r.Patch("/{doorId}", controller.UpdateDoor)
```

Comment:
patch > put cuz we aren't always updating every value and u dont need to pass every param to do an update, I set the Id as queryparam for all of these cuz its super annoying to parse

Reply:
Again, i'm sending the entire object from the frontend so the payload should contain all data you need (including Id). I think put is simpler here even if we aren't updated every field. I'm not going to argue if you want to use PATCH but from what I've read PUT is a simpler option less prone to mistakes.

Reply:
I don't have a strong opinion on this. Whatever is easiest to implement on the backend. I would assume that PUT would be easier to handle, but if it makes no difference then its fine.